src/lib/sqlite-command-versions.ts

```diff
@@ -88,9 +88,11 @@ async function checkForUpdate(): Promise< DistributionCheckResult > {
 	}
 }
 
-async function getCurrentSQLiteCommandVersion( versionFilePath: string ) {
+export async function getSQLiteCommandVersion( distributionPath: string ) {
 	try {
-		return ( await fs.readFile( versionFilePath, 'utf8' ) ).trim().replace( 'v', '' );
+		return ( await fs.readFile( path.join( distributionPath, VERSION_FILENAME ), 'utf8' ) )
+			.trim()
+			.replace( 'v', '' );
```

Comment:
I've observed that the two times we call this function we call to `semver.coerce`.

Should we move `semver.coerce` inside this function to always return a "valid" version?

Reply:
Yeah, I think it makes sense to return the semver value for simplicity.

Reply:
Suggestion applied in https://github.com/Automattic/studio/pull/434/commits/e1569af5cfe6baad9ff6b76ac6634ced2a45ec1b.